README.adoc

```diff
@@ -1,102 +1,405 @@
+That means the bulk of what you type are the words you want to communicate.
+You only add markup characters when you need to encode meaning that can’t otherwise be inferred.
+For example, a section title starts with a series of equals signs and an unordered list item begins with one or more asterisks.
+____
+
+The {project-name} lets you write AsciiDoc in a file on your local machine, then use your browser to view it, and make it easier to read and present.
+
+The file you are viewing now, is written in AsciiDoc and rendered to HTML by GitHub running asciidoctor on its servers.
+
+Many technical books, manuals and online help that you may use every day are written in AsciiDoc.
 
-To see your changes, click the `Reload` link in `about:debugging`.
+Help for writing asciidoc is available at {uri-asciidoc} and {uri-asciidoctor-docs} or search for example "asciidoc tables".
 
-== Options
+=== Example from this page
+The AsciiDoc used to write the previous section is shown below.
 
-The extension can be configured via an options page.
+[,asciidoc,subs=none]
+----
+== New to AsciiDoc
+AsciiDoc is a plain text markup language.
 
-To open the options page, in Chrome, Brave or Opera, right-click the extension icon and choose Options on the menu.
-You can also go to `chrome://extensions` and click the Options link.
+From the AsciiDoc {uri-asciidoc}[Home page]
+[QUOTE]
+____
+.Experience the magic of a lightweight markup language
+When you write in AsciiDoc, you use plain text.
+That means the bulk of what you type are the words you want to communicate.
+You only add markup characters when you need to encode meaning that can’t otherwise be inferred.
+For example, a section title starts with a series of equals signs and an unordered list item begins with one or more asterisks.
+____
 
-In Firefox, go to `about:addons` and click the More link or the Preferences button.
+The {project-name} lets you write AsciiDoc in a file on your local machine, then use your browser to view it, and make it easier to read and present.
+
+The file you are viewing now, is written in AsciiDoc and rendered to HTML by GitHub running asciidoctor on its servers.
+
+Many technical books, manuals and online help that you may use every day are written in AsciiDoc.
+
+Help for writing asciidoc is available at {uri-asciidoc} and {uri-asciidoctor-docs} or search for example "asciidoc tables".
+----
+
+=== Try it yourself
+To reproduce the examples shown in <<example>>, do the following:
+
+. Install the {project-name} using the guide
+. Copy and paste the AsciiDoc text below to a file called `file.adoc`
+. Open `file.adoc` from your browser
++
+.When viewing AsciiDoc files often
+TIP: You may change your computer settings to launch your web browser for `.adoc` files.
+. Toggle the {project-name} activition by clicking its extension icon in the browser bar.
+
+
+.Test file
+[,asciidoc,subs=none]
+....
+= AsciiDoc is Writing Zen
+My Email &lt;my.email@example.com&gt;
```

Comment:
There is no need to escape < and > in a verbatim block (like a list block). It will already be escaped. Just use < and > in the AsciiDoc source here.

Reply:
That's what I thought too. It disappeared when previewed in the GitHub editor. I will update and see how it looks out of the preview. Thank you for your comment.

Reply:
Updated. Current page. https://github.com/sturtison/asciidoctor-browser-extension/blob/docs/issue-648-refresh-docs-draft/README.adoc

![image](https://github.com/asciidoctor/asciidoctor-browser-extension/assets/13049926/fedfb44a-fd80-414c-bc4c-674ba772ce99)

Reply:
Take away subs=none. That's breaking it.